src/components/VmDisks/style.css

```diff
@@ -8,6 +8,7 @@
     list-style-type: none;
     padding: 0;
     margin-bottom: 0px !important;
+    display: table;
```

Comment:
I find this difficult to read and debug. This actually relies on generation of virtual, DOM invisible, `row` and `tr` elements as described at [Anonymous table objects][1] css spec. I find it much more clear to openly admit that we are actually creating complete css table and use appropriate `display` values for `li` (`table-row`) and `span (`table-cell`) elements.

[1]: https://www.w3.org/TR/CSS22/tables.html#anonymous-boxes